lib/assessment.rb

```diff
@@ -13,21 +13,21 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(:+)
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.any? { |numbers| (numbers % 2) != 0 }
```

Comment:
Correct. You could also use the `.odd?` method, e.g.
`numbers.any? {|num| num.odd?}`
or even
`numbers.any?(:odd?)`